web_poet/fields.py

```diff
@@ -16,6 +16,10 @@
 _FIELDS_INFO_ATTRIBUTE_WRITE = "_web_poet_fields_info_temp"
 
 
+def _fields_template():
+    return {"enabled": {}, "disabled": {}}
```

Comment:
An alternative approach is to declare something like:

```
_FIELDS_INFO_ATTRIBUTE_READ = "_web_poet_fields_info"
_FIELDS_INFO_ATTRIBUTE_WRITE = "_web_poet_fields_info_temp"
_FIELDS_INFO_ATTRIBUTE_READ_DISABLED = "_web_poet_fields_info_disabled"
_FIELDS_INFO_ATTRIBUTE_WRITE_DISABLED = "_web_poet_fields_info_temp_disabled"
```

But I think it could be unwieldly when more field-flags are introduced later on.